keys/keys.go

```diff
@@ -56,6 +56,16 @@ func NodeStatusKey(nodeID int32) proto.Key {
 	return MakeKey(StatusNodePrefix, encoding.EncodeUvarint(nil, uint64(nodeID)))
 }
 
+// NamespaceMetadataKey returns the key for the namespace.
+func NamespaceMetadataKey(namespace string) proto.Key {
+	return MakeKey(NamespaceMetadataPrefix, proto.Key(namespace))
+}
+
+// TableMetadataKey returns the key for the table in namespaceID.
+func TableMetadataKey(namespaceID uint32, tableName string) proto.Key {
+	return MakeKey(TableMetadataPrefix, encoding.EncodeUvarint(nil, uint64(namespaceID)), proto.Key("-"), proto.Key(tableName))
+}
```

Comment:
There are no need for separators such as `-` when using the ordered-code encodings. This should be written as:

```
  k := make([]byte, 0, len(TableMetadataPrefix) + len(tableName) + 10)
  k = append(k, TableMetadataPrefix...)
  k = encoding.EncodeUvarint(k, uint64(namespaceID))
  k = append(k, tableName...)
  return k
```